trunner/runners/common.py

```diff
@@ -39,9 +39,12 @@ def is_github_actions():
     return os.getenv('GITHUB_ACTIONS', False)
 
 
-def wait_for_dev(port, timeout=0):
+def wait_for_dev(port, timeout=0, wait_for_disappear=False):
     asleep = 0
 
+    if wait_for_disappear:
+        while os.path.exists(port):
```

Comment:
infinite loop is not a good idea, You should count timeout here also

Reply:
Finally not needed utility.